source/python.txt

```diff
@@ -28,3 +28,20 @@ to work with MongoDB in Python:
 - Use :doc:`Motor </motor>` for asynchronous Python applications.
 
 .. include:: /includes/university-m220p.rst
+
+Compatibility
+-------------
+
+You can connect your application to deployments hosted in the following 
+environments:
+
+- `{+atlas+} 
+  <https://www.mongodb.com/docs/atlas?tck=docs_server>`__: The fully
+  managed service for MongoDB deployments in the cloud
+- :ref:`MongoDB Enterprise <install-mdb-enterprise>`: The
+  subscription-based, self-managed version of MongoDB
+- :ref:`MongoDB Community <install-mdb-community-edition>`: The
+  source-available, free-to-use, and self-managed version of MongoDB
```

Comment:
Hi @carriecwk, thanks for this change! Since this PR was opened I merged #910, which added an include we could use here. Could you please replace the bullets with the include? With that change this is good to go from me, thanks!

```suggestion
.. include:: /includes/fact-environments.rst
```